src/redux/actionTypes/cellSets.js

```diff
@@ -74,7 +74,7 @@ const CELL_SETS_ERROR = `${CELL_SETS}/error`;
 export {
   CELL_SETS_LOADING, CELL_SETS_LOADED,
   CELL_SETS_CREATE,
-  CELL_SETS_UPDATE_PROPERTY, CELL_SETS_UPDATE_HIERARCHY, CELL_SETS_SET_SELECTED,
+  CELL_SETS_UPDATE_PROPERTY, CELL_SETS_REORDER, CELL_SETS_SET_SELECTED,
```

Comment:
Why are some actions in one line and other actions on multiple lines?

Reply:
I guess to have some kind of grouping? They need to be split up anyways